### DISCUSSION TITLE
Does Evo2 Learn Gene Function?

### DISCUSSION THREAD

**Discussion by GeorgiiVdovin** [ArcInstitute/evo2#145] [General]
Hello, thank you for the great work. I wanted to use Evo2 embeddings for comparative genomics, and was wondering: would information about gene function be learned by Evo2, and thus reflected in an embedding? E.g. if gene X is a DNA repair protein; and repairs DNA better in whale vs mouse; would the property of “better vs worse DNA repair” be reflected in the Evo2 embeddings of those genes, or would this feature be too specific? Have you seen any hints to that in your work?

**Comment by garykbrixi** [comment]
Hi @GeorgiiVdovin , thanks for your question! I think that embeddings should be useful for predicting function, suggested by the BRCA1 finetuning result. Related research has found that linear regression or other models on protein language model embeddings are able to predict specific functions from deep mutational scans, although the accuracy can be dependent on how many mutations away from the training data you are predicting. However it is an open question how much model embeddings capture specific properties such as DNA repair.
